src/Common.cpp

```diff
@@ -211,7 +211,7 @@ std::string NAS2D::join(std::vector<std::string> strs, char delim, bool skip_emp
 
 std::string NAS2D::join(std::vector<std::string> strs, bool skip_empty /*= true*/)
 {
-	auto acc_op = [](const std::size_t& a, const std::string& b) -> std::size_t { return a + static_cast<std::size_t>(1u) + b.size(); };
+	const auto acc_op = [](const std::size_t& a, const std::string& b) -> std::size_t { return a + static_cast<std::size_t>(1u) + b.size(); };
```

Comment:
My internal code quality ruleset built in to both Visual Assist and Visual Studio recommended declaring the the lambda variable `const`. Specifically because it was never written to, only read from.